resource/test/ignite_rules.json

```diff
@@ -1576,12 +2125,37 @@
         {
           "ApplicableFrom": null,
           "ApplicableTo": null,
-          "Identifier" : "Lab"
+          "Identifier": "Session Catalog (Mktg Site)"
+        },
+        {
+          "ApplicableFrom": null,
```

Comment:
All of these time bound session sets need to have the ApplicableFrom set at a minimum e.g "ApplicableFrom": "2016-09-27T00:00:00-05:00", leaving the ApplicableTo as null means that the effective time period of the session set runs until the end of the conference. So quite a few of these will need the ApplicableTo set as well.